src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -400,18 +400,11 @@ static ucs_status_t
 uct_rc_mlx5_get_cmd_qp(uct_rc_mlx5_iface_common_t *iface)
 {
     struct ibv_qp *qp;
-#ifdef HAVE_STRUCT_MLX5_SRQ_CMD_QP
-    iface->tm.cmd_wq.super.super.verbs.qp = NULL;
-    iface->tm.cmd_wq.super.super.verbs.rd = NULL;
-    iface->tm.cmd_wq.super.super.type     = UCT_IB_MLX5_OBJ_TYPE_LAST;
-    qp = uct_dv_get_cmd_qp(iface->rx.srq.verbs.srq);
```

Comment:
shall we also delete the function uct_dv_get_cmd_qp?

Reply:
Done. There's no corresponding function definition, just remove the function declaration.